InputDate/index.jsx

```diff
@@ -59,8 +140,9 @@ const InputDate = ({
             year: day.getFullYear(),
           });
         }
+        onDayClick()
       }}
-      selectedDays={!value ? null : new Date(value.year, value.month, value.day)}
```

Comment:
I believe this is important for `redux-form` style components: https://redux-form.com/6.0.0-alpha.4/docs/faq/customcomponent.md/

I think it would be more important for other repository to match this format.

A good example in our own repo of the implementation:

https://github.com/bufferapp/buffer-components/blob/master/InputStateless/index.jsx

Reply:
True! For now I found a simpler solution which runs with both formats. I believe in the future we should probably consider normalize everything and make sure the other calendars follow the input.value format, but I would do it on a separated task/PR to decrease the risk of crashing something, as things like the calendar in the composer are so important.

Reply:
Thank you for the context there :raised_hands: makes total sense to do this in stages to reduce risk :+1: